### PR TITLE
Adds lunchboxes

### PR DESCRIPTION
**Pull request by Just-a-Unity-Dev**
## About the PR

Adds lunchboxes

## Why / Balance

I think bringing lunchboxes to work is a nice idea

## Technical details

Adds respective lunchbox YML/texture paths

## Media

- [x] I have added screenshots/videos to this PR showcasing lunchboxes
image

## Breaking changes

Added lunchboxes.

**Changelog**

:cl: rain
- add: Added lunchboxes.

### CODE REVIEW

**Comment by Just-a-Unity-Dev** [comment]
FUCK

**Comment by Just-a-Unity-Dev** [comment]
Thanks git!

File: Resources/Prototypes/Loadouts/Miscellaneous/trinkets.yml [hunk 1]
```diff
@@ -29,6 +29,27 @@
...
+- type: loadout # CD
+  id: LunchboxYellow
+  equipment: LunchboxYellow
+
+- type: startingGear # CD
+  id: LunchboxYellow
+  storage:
+    back:
+    - LunchboxYellow
+
```
Comment (dffdff2423) [inline Resources/Prototypes/Loadouts/Miscellaneous/trinkets.yml:52]: Can you move this to it's own file under _CD?

**Review by dffdff2423 — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Loadouts/Miscellaneous/trinkets.yml (hunk 1 above)
Comment (Just-a-Unity-Dev) [reply Resources/Prototypes/Loadouts/Miscellaneous/trinkets.yml:52]: oops

Comment (Just-a-Unity-Dev) [reply Resources/Prototypes/Loadouts/Miscellaneous/trinkets.yml]: Resolved as of 21c635d8fd87c159edc8c194d345fd317068a91e

File: Resources/Prototypes/_CD/Entities/Objects/Misc/lunchboxes.yml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    size: 30
+  - type: MeleeWeapon
+    damage:
+      types:
+        Blunt: 3
+    soundHit:
+      path: "/Audio/Weapons/smash.ogg"
+  - type: Tag
+    tags:
+    - Toolbox
```
Comment (dffdff2423) [inline Resources/Prototypes/_CD/Entities/Objects/Misc/lunchboxes.yml:25]: Is there any reason for the toolbox tag? From the looks of things it is only used in cargo bounties (unless I missed something).

Comment (Just-a-Unity-Dev) [reply Resources/Prototypes/_CD/Entities/Objects/Misc/lunchboxes.yml:25]: This is what happens when I copy and paste code at 11pm

No real reason for it aside from me just forgetting to review it after I took it from the original toolbox thingy. I'll take it out in a few hours as I am going to bed

Comment (dffdff2423) [reply Resources/Prototypes/_CD/Entities/Objects/Misc/lunchboxes.yml:25]: No worries, get your sleep.

**Comment by DangerRevolution** [comment]
not sure if CD uses Bay sprites primarily (I just saw this PR while stalking the repo) but Delta has custom (and very nice) departmental lunchbox sprites if you wish, just to let you know :)

**Review by dffdff2423 — APPROVED** [review]
LGTM

**Pull request merged**